frontend/src/components/auth/components/form-controls.module.scss

```diff
@@ -7,3 +7,9 @@
   margin-top: 13px;
   margin-bottom: 22px;
 }
+
+.input-error {
+  border: 1px solid var(--input-error-color);
+  border-radius: 3px;
+  box-shadow: 0 0 3px 0 rgb(244 27 59 / 80%), 0 0 5px 0 rgb(244 27 59 / 50%);
```

Comment:
I guess you could take this color into a variable (and apply sass's color modification function to change opacity, if you like)